### ISSUE TITLE
TypeScript definition of layer misses params argument

### ISSUE THREAD

**Issue by heruan** [FortAwesome/Font-Awesome#15733]
**Describe the bug**
The TypeScript definition of the `layer` function misses the `params` argument, causing the compiler to raise an error:

https://github.com/FortAwesome/Font-Awesome/blob/1c1b102b2ba19518d77e60e97cf1b4ebf133bba0/js-packages/%40fortawesome/fontawesome-svg-core/index.d.ts#L13-L17

**To Reproduce**

This code:

```typescript
layer(push => push(faClock), { classes: ['fa-fw'] });
```
raises a compiler error:

```
Expected 1 arguments, but got 2.
```

**Expected behavior**

It compiles without error.

**Version and implementation**
Version: 5.11.2

- [x] SVG with JS
- [ ] Web Fonts with CSS
- [ ] SVG Sprites
- [ ] On the Desktop

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and thanks for reporting this.

@robmadole any chance to take a look here?

**Comment by robmadole** [comment]
We'll get this fixed in 5.12.

**Comment by tagliala** [comment]
I think this has been fixed in 5.12, please comment if it is still an issue
